### PR TITLE
Sb 1095 add tu handler

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: annotations/component-annotations/src/it/issue-71/src/main/java/io/ap4k/examples/component/Main.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 import io.ap4k.kubernetes.annotation.Env;
 
 @CompositeApplication(name = "hello-world", exposeService = true, envVars = @Env(name = "key1", value = "val1"))
-@Link(name = "hello-world", targetcomponentname = "target", envVars = @Env(name = "key1", value = "val1"))
+@Link(name = "hello-world", componentname = "target", envVars = @Env(name = "key1", value = "val1"))
```
Comment (metacosm) [inline annotations/component-annotations/src/it/issue-71/src/main/java/io/ap4k/examples/component/Main.java]: Can we use `componentName` instead, please?

File: annotations/component-annotations/src/main/java/io/ap4k/component/annotation/Link.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
 @Retention(RetentionPolicy.SOURCE)
 public @interface Link {
   String name() default "";;
-  String targetcomponentname();
+  String componentname();
```
Comment (metacosm) [inline annotations/component-annotations/src/main/java/io/ap4k/component/annotation/Link.java]: Same.

File: annotations/component-annotations/src/main/java/io/ap4k/component/handler/LinkHandler.java [hunk 3]
```diff
@@ -49,7 +54,7 @@ private Link createLink(LinkConfig config) {
       .withName(config.getName())
       .endMetadata()
       .withNewSpec()
-      .withName(config.getName())
+      .withComponentName(config.getComponentname())
```
Comment (metacosm) [inline annotations/component-annotations/src/main/java/io/ap4k/component/handler/LinkHandler.java]: Same.

File: annotations/component-annotations/src/it/issue-71/src/main/java/io/ap4k/examples/component/Main.java (hunk 1 above)
Comment (aureamunoz) [reply annotations/component-annotations/src/it/issue-71/src/main/java/io/ap4k/examples/component/Main.java]: yes, sure!

**Pull request merged**